src/RBACHandler.ts

```diff
@@ -42,19 +43,24 @@ export class RBACHandler implements Authorization {
         const decoded = decode(request.accessToken, { json: true }) || {};
         const groups: string[] = decoded['cognito:groups'] || [];
 
-        if (request.exportType) {
-            return this.isExportAllowed(groups, request.exportType);
+        if (request.export) {
+            const requesterUserId = this.getRequesterUserId(request.accessToken);
+            return this.isExportAllowed(groups, requesterUserId, request.export);
         }
         return this.isAllowed(groups, request.operation, request.resourceType);
     }
 
-    private isExportAllowed(groups: string[], exportType: ExportType): boolean {
+    private isExportAllowed(groups: string[], requesterUserId: string, exportAuth: ExportAuth): boolean {
+        const { jobRequesterUserId, operation, type } = exportAuth;
+        if (['get-status', 'cancel-export'].includes(operation)) {

```

Comment:
When we have import will 'get-status' be used with the same 'operation' name?

Reply:
We can use `get-status-import` and `get-status-export`